proto/osmosis/streamswap/v1/tx.proto

```diff
@@ -0,0 +1,141 @@
+syntax = "proto3";
+package osmosis.streamswap.v1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v10/x/streamswap/types";
+option (gogoproto.goproto_getters_all) = false;
+
+service Msg {
+  // CreateSale creates new token sale. Anyone can create a new sale.
+  // params.SaleBond OSMO will be charged as a bond (returned in FinalizeSale)
+  // to avoid spams.
+  // The sale follows the streamswap functionality explained in the
+  // x/launchapd/spec
+  rpc CreateSale(MsgCreateSale) returns (MsgCreateSaleResponse);
+
+  // Subscribe to a token sale. Any use at any time before the sale end can join
+  // the sale by sending `token_in` to the Sale through the Subscribe msg.
+  // During the sale, user `token_in` will be automatically charged every
+  // epoch to purchase `token_out`.
+  rpc Subscribe(MsgSubscribe) returns (google.protobuf.Empty);
+
+  // Withdraw sends back `amount` of unspent tokens_in to the user.
+  // If `amount` is empty, it will default to all unspent tokens.
+  // User can do it any time unless his deposit is empty.
+  rpc Withdraw(MsgWithdraw) returns (google.protobuf.Empty);
+
+  // ExitSale withdraws (by a user who subscribed to the sale) purchased
+  // tokens_out from the pool and remained tokens_in. Returns error if it's
```

Comment:
I think the comment that it returns error can be misleading for the proto documentations

Reply:
updated the comment using "Must".